templates/spec/rails_helper.rb

```diff
@@ -23,4 +23,19 @@
   config.before(:each, type: :system, js: true) do
     driven_by :selenium_chrome_headless
   end
+
+  config.after(:each, type: :system, js: true) do
```

Comment:
i'd consider to leave that piece of code and replace the one above  in ruby_on_rails/rspec.md with a link so we only have to update one.